app/src/main/java/org/kiwix/kiwixmobile/wifi_hotspot/WifiHotspotManager.java

```diff
@@ -0,0 +1,89 @@
+package org.kiwix.kiwixmobile.wifi_hotspot;
+
+import android.content.Context;
+import android.net.wifi.WifiConfiguration;
+import android.net.wifi.WifiManager;
+import android.os.Build;
+import android.os.Handler;
+import android.util.Log;
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import org.kiwix.kiwixmobile.webserver.ServerStateListener;
+
+/**
+ * WifiHotstopManager class makes use of the Android's WifiManager and WifiConfiguration class
+ * to implement the wifi hotspot feature.
+ * Created by Adeel Zafar on 28/5/2019.
+ */
+
+public class WifiHotspotManager {
+  private final WifiManager wifiManager;
+  private final Context context;
+  WifiManager.LocalOnlyHotspotReservation hotspotReservation;
+  boolean isHotspotEnabled;
+  WifiConfiguration currentConfig;
```

Comment:
this can be local, doesn't need to be a field

Reply:
Made it a local field. But its being used more than once. Shall we still remove it?
df0a5e6eb11791f38114646c735bdfcfd8c6df3c

Reply:
Unify declaration and assignment. Also what you did was change a `field` to  a `local variable`.
The variable is used in 2 places so it is fine to keep it a variable, though one of those places is a log and I generally avoid logging things. Up to you

Reply:
Fixed. 9143af7aa456b040a937b23f846361e20193ed78